### DISCUSSION TITLE
Unrecognized function or variable 'ps_parms_default'

### DISCUSSION THREAD

**Discussion by euyurioliveira** [alexisInSAR/EZ-InSAR#86] [Help]
Hello everybody
Hello @alexisInSAR 

I have a problem when trying to prepare for the PSI analysis with StaMPS. Apparently I have a problem with the 'ps_parms_default' function that is not being recognized. Also, do you have a suggestion as to what could be the cause of the problem? Below are the images.

![Captura de tela de 2024-03-19 20-55-32](https://github.com/alexisInSAR/EZ-InSAR/assets/108848153/32d5f37a-849b-4a90-9f86-80baf8f10f6f)

Thank you all in advance for your help.

**Comment by alexisInSAR** [comment]
Hi, 

I think your StaMPS directory is not in the MATLAB path variable. You can add it by clicking (right) on the StaMPS directory. 

Alexis
